engine/schema/src/main/java/com/cloud/storage/GuestOSVO.java

```diff
@@ -113,7 +113,7 @@ public Date getRemoved() {
     }
 
     @Override
-    public boolean isUserDefined() {
+    public boolean getIsUserDefined() {
```

Comment:
not a good rename

Reply:
agree, but i think it works if field name is _userDefined_

https://github.com/apache/cloudstack/blob/41d25d54a79c21877b4c493261ab6a60251b9f34/framework/db/src/main/java/com/cloud/utils/db/SearchBase.java#L484-L486

https://github.com/apache/cloudstack/blob/41d25d54a79c21877b4c493261ab6a60251b9f34/framework/db/src/main/java/com/cloud/utils/db/SqlGenerator.java#L83-L84